ccd-definition/CaseEventToComplexTypes/placement/Placement-prepare-for-hearing.json

```diff
@@ -1,63 +0,0 @@
-[
-  {
-    "LiveFrom": "01/01/2017",
-    "ID": "Placement",
-    "CaseEventID": "placement-PREPARE_FOR_HEARING",
```

Comment:
there is still 

  {
   ......
    "CaseEventID": "placement-PREPARE_FOR_HEARING",
    "UserRole": "[LASOLICITOR]",
    "CRUD": "CRU"
  },